src/main/java/com/iridium/iridiumskyblock/commands/BlockValueCommand.java

```diff
@@ -49,7 +59,7 @@ public void execute(CommandSender sender, String[] arguments) {
     public List<String> onTabComplete(CommandSender commandSender, org.bukkit.command.Command command, String label, String[] args) {
         // We currently don't want to tab-completion here
         // Return a new List so it isn't a list of online players
-        return Collections.emptyList();
+        return Arrays.asList("BLOCK", "SPAWNER");
```

Comment:
Don't hardcode this, use Enum#values()#stream()#map(Enum::name)#collect(Collectors.toList())